.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+# This is a basic workflow to help you get started with Actions
+name: CI
+
+# Controls when the action will run. Triggers the workflow on pull request
+# events but only for the master branch
+on:
+  pull_request:
+    branches: [ master, 5.X.X, 4.X.X, 3.X.X, 2.X.X ]
```

Comment:
wenn das für all diese Branches gelten soll, muss die File zuerst in 2.X.X rein

Reply:
```suggestion
    branches: [ 5.X.X ]
```